core/mutex.c

```diff
@@ -60,11 +59,59 @@ int mutex_lock(mutex_t *mutex)
         thread_add_to_list(&mutex->queue, me);
     }
 
-
     irq_restore(irq_state);
     thread_yield_higher();
     /* We were woken up by scheduler. Waker removed us from queue. */
-    return 0;
+}
+
+void mutex_lock(mutex_t *mutex)
+{
+    unsigned irq_state = irq_disable();
+
+    DEBUG("PID[%" PRIkernel_pid "]: Mutex in use.\n", thread_getpid());
```

Comment:
What does this mean? Maybe rephrase: "pid x: Locking mutex %p"?

Reply:
I made the debug output more consistent and just use the function names, which both describe quite well what is done and how to find the corresponding code.